cypress/e2e/validation.cy.js

```diff
@@ -0,0 +1,59 @@
+const invEmail = "invalid123@noroff.no";
+const invPassword = "passwordinvalid";
+
+describe("Login validation", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    cy.wait(1000);
+    cy.get("#registerModal button[type=button]").contains("Login").click();
+    cy.wait(2000);
+  });
+
+  it("should deny login with invalid info", () => {
+    cy.get("input[type='email']:visible").type(invEmail, {
+      delay: 500,
+    });
+    cy.get("input[type='password']:visible").type(invPassword, {
+      delay: 500,
+    });
+    cy.get("button[type=submit]").contains("Login").click();
+    cy.wait(1000);
```

Comment:
What does this error look like, how could you prove it?

Reply:
The issue is now fixed.